### PR TITLE
Moved from hardware dependent feature to performance dependent

### PR DESCRIPTION
**Pull request by radujipa**
Related to https://github.com/KanoComputing/Kerberos/issues/4

### CODE REVIEW

**Comment by Ealdwulf** [comment]
:+1:

File: bin/kano-video-browser [hunk 1]
```diff
@@ -7,12 +7,11 @@
 #
 
 from kano.gtk3.kano_dialog import KanoDialog
-from kano.utils import run_cmd, get_home, is_model_2_b
+from kano.utils import run_cmd, has_min_performance, RPI_2_B_SCORE
 from kano.logging import logger
 from kano_settings.system.advanced import get_parental_level
 
 from gi.repository import Gtk
-import shutil
```
Comment (tombettany) [inline bin/kano-video-browser]: Was this just unused?

**Comment by tombettany** [comment]
:+1:

File: bin/kano-video-browser (hunk 1 above)
Comment (radujipa) [inline bin/kano-video-browser]: Yup

**Pull request merged**